example/cnn_chinese_text_classification/text_cnn.py

```diff
@@ -20,20 +20,19 @@
 
 # -*- coding: utf-8 -*-
 
-import sys
-import os
+import sys, os
 import mxnet as mx
 import numpy as np
 import argparse
 import logging
-
 import time
 
 from mxnet import random
 from mxnet.initializer import Xavier, Initializer
 
 import data_helpers
 
+print("Training logs are generated in ./cnn_text_classification.log")
```

Comment:
nitpick: There is only one log file. Isn't it? If yes, then this may be more appropriate "Training log is generated at ./cnn_text_classification.log" or "Training log can be found at cnn_text_classification.log"